src/components/segments/VimeoVideo.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default class Video extends React.Component {
+  render() {
+    return (
+    	<div className="vimeo-video">
+      		<iframe src="https://player.vimeo.com/video/126984194" width="500" height="281" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
```

Comment:
I am not familiar with anything like that. But I will take a look at your suggestion.